ext/oberon/psa/core/library/psa_crypto.c

```diff
@@ -5205,6 +5232,13 @@ psa_status_t psa_crypto_init(void)
     }
 #endif /* PSA_CRYPTO_STORAGE_HAS_TRANSACTIONS */
 
+#if defined(CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK) && !defined(CONFIG_BUILD_WITH_TFM)
+    status = hw_uique_key_provisioning();
+    if (status != PSA_SUCCESS) {
+        goto exit;
+    }
+#endif /* CONFIG_TRUSTED_STORAGE_BACKEND_AEAD_KEY_DERIVE_FROM_HUK && !CONFIG_BUILD_WITH_TFM */
+
```

Comment:
Did you consider putting this in

https://github.com/nrfconnect/sdk-nrf/pull/14231/files#diff-aa91a034446fc11854bec771f103e132d0657e3777a0e5fdca9c0bca30925d51R5204

?

Then we would only need to make changes to crypto driver wrappers, and not the core.

Reply:
makes sense! moved